src/Provider/Azure.php

```diff
@@ -28,11 +28,22 @@ class Azure extends AbstractProvider
      *
      * @var string
      */
-    protected $urlAuthorize = 'https://login.microsoftonline.com';
+    protected $urlAuthorize;
 
     /** @var string */
     protected $urlResourceOwnerDetails = 'https://graph.microsoft.com/v1.0/me';
 
+    /**
+     * @param array $options
+     * @param array $collaborators
+     * @param string $urlAuthorize Base url for authorization.
+     */
+    public function __construct(array $options = [], array $collaborators = [], string $urlAuthorize = 'https://login.microsoftonline.com')
```

Comment:
Just letting clear that this syntax could be much cleaner, using constructor property promotion, but I saw that the package composer file requires support to PHP 7 (property promotion is only supported by php 8.x).